### DISCUSSION TITLE
A few words about the upcoming testnet.

### DISCUSSION THREAD

**Discussion by ramuchi** [NillionNetwork/discussions#3] [General]
Hello! Team Nillion!
Is there any information about the upcoming testnet that requires validators? 
If so, will there be a form or how to apply to participate? thanks!

**Comment by oceans404** [comment]
gm @ramuchi! Nillion nodes are currently permissioned, but we will eventually make node operation permissionless. Keep an eye on this page:  https://docs.nillion.com/running-a-node and our socials

**Comment by 0xConstruct** [comment]
Thanks @oceans404 is there an ETA on nodes?
